### PR TITLE
Add ASIO rights statements

### PR DESCRIPTION
**Pull request by gilgongo**
# Changes

Add a logo and rights statement to the footer. Add a statement about binary distribution on the Windows download.

BTW couldn't work out how to show the logo on the home page only, so it's in the "global" footer. I've also removed the "wiki" footer so that now all software rights are in the "global" footer, and moved the docs CC statement to the nav for clarity.

## Does this need translation?

- [x] Yes 
- [ ] No 

## Related issues

https://github.com/jamulussoftware/jamulus/issues/2417

### CODE REVIEW

**Comment by ann0see** [comment]
Isn’t it enough to put that on the Windows install page (Windows is the only place where ASIO is used)

**Comment by gilgongo** [comment]
Not according to Steinberg's rather long and complicated terms as far as I can tell. And in any case, I'd rather not push our luck :-)

**Comment by ann0see** [comment]
ok. so we should ask our designers how to include it...

**Comment by gilgongo** [comment]
Just put it in the footer. I forgot to add the image to my commit will do that later

File: wiki/en/Installation-for-Windows.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ Upgrading? You may want to [back up your configuration](Software-Manual#backing-
 1. **Run Jamulus**. Now you should be able to use Jamulus just like any other application.
 1. **Set up your sound card**. When you're done, you need to set up your audio hardware. And if you are using ASIO4ALL, have a look at how to set that up below.
 
+_You are permitted to redistribute this binary without acquiring a licence agreement from Steinberg._
```
Comment (ann0see) [inline wiki/en/Installation-for-Windows.md]: I thought users were not permitted to do that? I didn't participate in the licensing discussion, so probably it's better to leave it up to the people who did...

File: wiki/en/misc/general.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 footer:
-  copyright:         Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/jamulussoftware/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a>
+  copyright:         Software by <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> and <a href="https://github.com/jamulussoftware/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributors</a> - want to <a href="/wiki/Contribution">get involved?</a> <p> <img alt="ASIO logo" src="/assets/img/asio.jpg" id="asio" loading="lazy"> <br><em>ASIO is a trademark and software of Steinberg Media Technologies GmbH</em>
```
Comment (ann0see) [inline wiki/en/misc/general.yml:2]: I'm not sure if this looks ok in the footer. I'd rather create another FOX-CSS container which maybe also links to JACK...

File: _includes/general/nav.html [hunk 3]
```diff
@@ -30,5 +30,6 @@ <h3>{{ item.title }}</h3>
       {% endfor %}
       {% endif %}
     </ul>
+    <span style="color:#696969;font-size:12px">This documentation is <a href="https://creativecommons.org/licenses/by-sa/4.0/deed.en">CC BY-SA</a></span>
```
Comment (ann0see) [inline _includes/general/nav.html:33]: Please DO NOT hardcode css here. Generate a class/id and put it into a CSS file.

File: _layouts/wiki.html [hunk 4]
```diff
@@ -15,7 +15,6 @@
     <div id="maincontainer" class="fx-col-100-xs fx-col-50-m">
       <noscript class="red"><strong>{{ site.data.general.tNoJSEnabled }}</strong></noscript>
       <main lang="{{ page.lang }}">{{ content }}</main>
-      <div id="ftext" class="fx-txt-center">
```
Comment (ann0see) [inline _layouts/wiki.html:18]: Only one div removed? Shouldn't the closing div also be removed (including the CSS id)?

File: wiki/en/misc/footertext.md [hunk 5]
```diff
@@ -1,6 +1,2 @@
```
Comment (ann0see) [inline wiki/en/misc/footertext.md:1]: If this file is no longer used, we should remove it totally.

**Review by ann0see — COMMENTED** [review]
If it's just about adding the logo, I think we should add it somewhere in a html template.

**Comment by ann0see** [comment]
This is the homepage layout: https://github.com/jamulussoftware/jamuluswebsite/blob/release/_layouts/mainhomepage.html

**Comment by gilgongo** [comment]
Apologies, this is only a draft. Come back later and I'll have done it a bit better maybe 🙂

**Comment by ann0see** [comment]
Ok. No problem. Just ping me if you're finished.

File: wiki/en/Installation-for-Windows.md (hunk 1 above)
Comment (gilgongo) [reply wiki/en/Installation-for-Windows.md]: Hehe. Forgot the "not"  Wonderful. :-)

**Comment by gilgongo** [comment]
Hm. Conflicts. Maybe I should re-submit this.

**Comment by ann0see** [comment]
Re-basing is the best (git rebase -i). 

Maybe https://m.youtube.com/watch?v=f1wnYdLEpgI  or https://m.youtube.com/watch?v=_UZEXUrj-Ds helps getting used to the term.

**Comment by gilgongo** [comment]
Those videos look pretty complicated...! I don't know what the consequences of getting it wrong might be, so I'll be a coward and close to re-submit later think :-)

**Pull request closed without merging**